docs/Ariba-connector.md

```diff
@@ -0,0 +1,25 @@
+# Ariba Connection
+
+Description
+-----------
+Use this connection to access data in Salesforce.
+
+Properties
+----------
+**Ariba Client ID (M)**: Ariba Client ID.
+
+**Ariba Client Secret (M)**: Ariba Client Secret.  
+
+**Ariba API Key (M)**: Ariba API Key.
+
+**API Endpoint (M)**: Base path of the Ariba API, for example, https://sandbox.api.sap.com. 
```

Comment:
When I view this .md file, the URL has a link? Can the link be removed since it's not a real url?